UTRA-ART/Caffeine
# CAFF-71 - Fixed sensor out of bounds

-Fixes the warning that appears when the sensor is outside of the costmap bounds
-Test: 
1. Run gazebo
2. Run rviz
3. Run move_base
4. roslaunch load_waypoints load_waypoints.launch
-Expectation: the robot should move to the different waypoints without any warnings about the sensor being outside of costmap boundaries.
